server/plugin.go

```diff
@@ -56,6 +58,11 @@ type Client interface {
 	GetUser(user *model.User) (*zoom.User, *zoom.AuthError)
 }
 
+type ZoomPluginAPI interface {
+	GetZoomSuperUserToken() (*oauth2.Token, error)
+	SetZoomSuperUserToken(*oauth2.Token) error
+}
```

Comment:
This type should just be defined in the `server/zoom` package. What's great about Go is that the `server` package does not need to know about the interface itself. The `Plugin` struct implicitly implements the interface, and that's all the `server/zoom` package cares about when it receives that argument.

Reply:
I have moved this now and renamed the interface to PluginAPI to fix a lint issue.